client/app/counter/counter.service.ts

```diff
@@ -12,10 +12,10 @@ export class CounterService {
         private http: HttpClient
     ) { }
 
-    getCounter(): Observable<any> {
+    getOrders(): Observable<any> {
```

Comment:
I think for these order related API actions there should be a new `OrderService`?

Reply:
Yeah, i need to implement it. just noticed the changes from dev